.github/workflows/android-test.yml

```diff
@@ -11,9 +11,8 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        # Currently, lower terminals fail. ref: https://github.com/Kotlin/kotlinx-datetime#using-in-your-projects
         # Currently, no API 30 virtual image
-        api-level: [26, 27, 28, 29]
+        api-level: [21, 23, 26, 29]
```

Comment:
Passed in my forked repository.

https://github.com/hkusu/conference-app-2021/actions/runs/657985468